backend/internal/userschema/constants/schemaconstants.go

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// Package constants defines schema constants for user schema management operations.
+package constants
+
+// JSON Schema type constants.
```

Comment:
Is it intentional to define these constants in two files?
`userschema/constants/schema/types.go` and `userschema/constants/schemaconstants.go`

Reply:
no.. will be move to the model.userschema as this is more closer to the model constructs..